### PR TITLE
PD: add description of batch in evict-leader-scheduler config

### PR DESCRIPTION
**Pull request by niubell**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add a new config item: batch in evict-leader-scheduler from v8.3.0, so add the instructions for use.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -1322,6 +1322,11 @@ scheduler config balance-hot-region-scheduler  // 显示 balance-hot-region 调
 
     当一个 `evict-leader-scheduler` 的所有 store 配置都被移除后，该调度器也会自动被移除。
 
+- `batch`可以用来调整单次调度过程中生成 `Operator` 的数量，`batch` 值越大，调度速度越快。在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值，示例如下：
+    ```bash
+    scheduler config evict-leader-scheduler set batch 10 // 设置 batch 值为 10
```
Comment (HuSharp) [inline pd-control.md:1327]: - 需要提及设置 batch 的范围在 [1, 10] 吗？
- 需要提及默认值为 3 吗？
- 需要提及支持版本号吗？

Comment (niubell) [reply pd-control.md:1327]: 已添加默认值及取值范围说明，版本号感觉会在对应文档版本及 release note 里提及，不需要在配置项上专门说明。

**Review by HuSharp — APPROVED** [review]

File: pd-control.md [hunk 2]
```diff
@@ -1322,6 +1322,11 @@ scheduler config balance-hot-region-scheduler  // 显示 balance-hot-region 调
 
     当一个 `evict-leader-scheduler` 的所有 store 配置都被移除后，该调度器也会自动被移除。
 
+- `batch`可以用来调整单次调度过程中生成 `Operator` 的数量，`batch` 值越大，调度速度越快，默认值为：3，取值范围：[1,10]。在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值，示例如下：
+    ```bash
```
Comment (qiancai) [inline pd-control.md:1326]:
````suggestion
- 在已有 `evict-leader-scheduler` 时，可以使用 `set batch` 子命令修改 `batch` 值。其中，`batch` 用于调整单次调度过程中生成 `Operator` 的数量，默认值为 `3`，取值范围为 `[1, 10]`。`batch` 值越大，调度速度越快。

    ```bash
````

File: pd-control.md [hunk 3]
```diff
@@ -1322,6 +1322,12 @@ scheduler config balance-hot-region-scheduler  // 显示 balance-hot-region 调
 
     当一个 `evict-leader-scheduler` 的所有 store 配置都被移除后，该调度器也会自动被移除。
 
+- 在已有 `evict-leader-scheduler` 时，可以使用 `set batch` 子命令修改 `batch` 值。其中，`batch` 用于调整单次调度过程中生成 `Operator` 的数量，默认值为 `3`，取值范围为 `[1, 10]`。`batch` 值越大，调度速度越快。
```
Comment (qiancai) [inline pd-control.md:1325]:
```suggestion
- 在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值。其中，`batch` 用于调整单次调度过程中生成 `Operator` 的数量，默认值为 `3`，取值范围为 `[1, 10]`。`batch` 值越大，调度速度越快。
```

File: pd-control.md [hunk 4]
```diff
@@ -1322,7 +1322,8 @@ scheduler config balance-hot-region-scheduler  // 显示 balance-hot-region 调
 
     当一个 `evict-leader-scheduler` 的所有 store 配置都被移除后，该调度器也会自动被移除。
 
-- `batch`可以用来调整单次调度过程中生成 `Operator` 的数量，`batch` 值越大，调度速度越快，默认值为：3，取值范围：[1,10]。在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值，示例如下：
+- 在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值。其中，`batch` 用于调整单次调度过程中生成 `Operator` 的数量，默认值为 `3`，取值范围为 `[1, 10]`。`batch` 值越大，调度速度越快。
```
Comment (qiancai) [inline pd-control.md:1325]:
```suggestion
- 在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值。其中，`batch` 用于调整单次调度过程中生成的 `Operator` 数量，默认值为 `3`，取值范围为 `[1, 10]`。`batch` 值越大，调度速度越快。
```

**Review by qiancai — APPROVED** [review]

File: pd-control.md [hunk 5]
```diff
@@ -1322,6 +1322,12 @@ scheduler config balance-hot-region-scheduler  // 显示 balance-hot-region 调
 
     当一个 `evict-leader-scheduler` 的所有 store 配置都被移除后，该调度器也会自动被移除。
 
+- 在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值。其中，`batch` 用于调整单次调度过程中生成的 `Operator` 数量，默认值为 `3`，取值范围为 `[1, 10]`。`batch` 值越大，调度速度越快。
```
Comment (Oreoxmt) [inline pd-control.md:1325]:
```suggestion
- 在已有 `evict-leader-scheduler` 时，使用 `set batch` 子命令修改 `batch` 值。其中，`batch` 用于调整单次调度过程中生成的 Operator 数量，默认值为 `3`，取值范围为 `[1, 10]`。`batch` 值越大，调度速度越快。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**